wandb/sdk/artifacts/artifact.py

```diff
@@ -1268,66 +1205,47 @@ def add_reference(
     ) -> Sequence[ArtifactManifestEntry]:
```

Comment:
@ibindlish - is Sequence a W&B specific object? Does this represent an Artifact Sequence?

Reply:
No, this has nothing to do with artifact sequences. A python `Sequence` is just an ordered set that is immutable.